build.gradle

```diff
@@ -46,7 +46,7 @@ plugins {
   id 'org.scoverage' version '8.0.3' apply false
   // Updating the shadow plugin version to 8.1.1 causes issue with signing and publishing the shadowed
   // artifacts - see https://github.com/johnrengelman/shadow/issues/901
```

Comment:
I think this can be removed now.

Reply:
The Comment has been removed as suggested